### ISSUE TITLE
ranked solo quitter debuff not appearing 

### ISSUE THREAD

**Issue by barfyman92** [WarEmu/WarBugs#19255]
I qued up and it popped then i crashed and now i cant reque (it says i have quitter but no debuff is shown)

iirc it is 30 minute quitter for leaving or not taking ranked solo so i just logged off and am now typing this, sucks because i had popped potions etc

**Comment by lilimx** [comment]
hey,

hm it's actual a 20 min quitter debuff.
you've tried to /rel when you had the debuff?

**Comment by barfyman92** [comment]
no i did not, i did relog though to see it it would fix it but it didn't

**Comment by lilimx** [comment]
hey,

but you are able to queue again now?

**Comment by barfyman92** [comment]
Yes works fine now, i worded myself poorly in the OP sry about that. 

I crashed on the loading screen to the scenario and when i logged back in i couldn't que ( errormessage in the chatbox said i had quitter ) but i could not see that i had any quitter debuff icon visisble anywher eon my screen. 

I logged out to take a break and when i got back to the game it worked.

The bug is that I crashed and got quitter, AND it was invisible. Should not work like that afaik, crashes should send you back into the game not give you quitter.
